app/views/CreditsSummary.scala.html

```diff
@@ -86,6 +86,10 @@
     }
 }
 
+@getMoneyInYourAccountSectioName(isAgent: Boolean) = @{
```

Comment:
isAgent is global variable, so it use is redundant in scope of this help function;

Reply:
done thanks